web/js/roomController.js

```diff
@@ -856,6 +856,21 @@ PhoneNumberController, ResizeSensor */
     };
 
     var connect = otHelper.connect.bind(otHelper, sessionInfo);
+    
+    var waitForConnectionCount = function() {
+      return new Promise(function (resolve) {
+        if (!maxUsersPerRoom) {
+          return resolve();
+        }
+        return setTimeout(function () {
+          if (numUsrsInRoom > maxUsersPerRoom) {
+            Utils.sendEvent('roomController:meetingFullError');
+            return;
+          }
+          resolve();
+        }, 500);
```

Comment:
If anything, I think we can make this shorter than .5 seconds. But I haven't tested enough, and I don't think that's too long to wait for `connectionCreated` events to come in after connecting to the session.

Reply:
is there any possibility this could take longer than 500 milis?

Reply:
I tried it locally and it worked perfectly. im asking just in case